packages/statistics-module/src/index.js

```diff
@@ -0,0 +1,3 @@
+export default function statisticsModule() {
+  // TODO
+}
```

Comment:
I think it would be better if we follow the same format as `links-module` and just leave it as 

`module.exports = {}`

at least for now.